samples/java-customer-registry-kafka-quickstart/README.md

```diff
@@ -30,9 +30,11 @@ It also contains the configuration to start a local Kafka broker and a Zookeeper
 To start the proxy, the Kafka broker, and the Zookeeper, run the following command from the root directory:
 
 ```shell
-docker compose -f kafka/docker-compose.yml up
+docker-compose -f kafka/docker-compose.yml rm -f && docker-compose -f kafka/docker-compose.yml up
```

Comment:
```suggestion
docker compose -f kafka/docker-compose.yml rm -f && docker compose -f kafka/docker-compose.yml up
```
It's a command in `docker` now, right?

Reply:
oh, interesting. I did not know, but it makes sense. I was always using `docker-compose` but will change it now.